frontend/public/components/utils/timestamp.jsx

```diff
@@ -75,9 +75,12 @@ export class Timestamp extends SafetyFirst {
     }
 
     const timeAgo = now - mdate;
-    if (timeAgo < 630000) { // 10.5 minutes
+    if (timeAgo < 630000 && !this.props.noTooltip) { // 10.5 minutes
       return dateTime.fromNow(mdate);
     }
+    if (this.props.noTooltip) {
```

Comment:
In this context, why add the toggle to turn off the tooltip?

Reply:
Also, why bind the ideas of `noTooltip` to `fromNowSimple`?   Just for clarity.

Reply:
The event frequency line is a special case usage of Timestamp. I wanted to reuse most of Timestamp but needed to skip returning the tooltip, icon, etc. and to display different text than it usually does in this case. (8 minutes instead of 8 minutes ago.)

If you have naming suggestions that will get this across better, let me know.

Reply:
Maybe `props.simple`. I wouldn't call it `noTooltip` if it has other effects.

Reply:
You could even have two props: `props.omitSuffix` to leave off the suffix and `props.simple` to disable the icon and tooltip.